apps/web/src/components/Profile/components/GovernmentInformation/Display.tsx

```diff
@@ -3,42 +3,60 @@ import { useNavigate } from "react-router";
 
 import { commonMessages } from "@gc-digital-talent/i18n";
 import { empty } from "@gc-digital-talent/helpers";
+import { FragmentType, getFragment, graphql } from "@gc-digital-talent/graphql";
 
 import { wrapAbbr } from "~/utils/nameUtils";
 import profileMessages from "~/messages/profileMessages";
 import useRoutes from "~/hooks/useRoutes";
 import FieldDisplay from "~/components/FieldDisplay/FieldDisplay";
 
-import { PartialUser } from "./types";
 import EmailVerificationStatus from "../EmailVerificationStatus";
 
+export const GovernmentInformationDisplay_Fragment = graphql(/** GraphQL */ `
+  fragment GovernmentInformationDisplay on User {
+    isGovEmployee
+    hasPriorityEntitlement
+    priorityNumber
+    govEmployeeType {
+      value
+      label {
+        localized
+      }
+    }
+    department {
+      id
+      name {
+        localized
+      }
+    }
+    currentClassification {
+      group
+      level
+    }
+    workEmail
+    isWorkEmailVerified
+  }
+`);
+
 interface DisplayProps {
-  user: PartialUser;
+  query: FragmentType<typeof GovernmentInformationDisplay_Fragment>;
   showEmailVerification?: boolean;
   readOnly?: boolean;
 }
 
 const Display = ({
-  user: {
-    isGovEmployee,
-    department,
-    govEmployeeType,
-    currentClassification,
-    hasPriorityEntitlement,
-    priorityNumber,
-    workEmail,
-    isWorkEmailVerified,
-  },
+  query,
   showEmailVerification = false,
   readOnly = false,
 }: DisplayProps) => {
   const intl = useIntl();
   const navigate = useNavigate();
   const routes = useRoutes();
+  const user = getFragment(GovernmentInformationDisplay_Fragment, query);
```

Comment:
You could make smaller diffs if you deconstructed user like you did in most of the other Display components.

Reply:
Ahh yes, thanks for catching that. I regretted this decison and forgot to come back and fix these :sweat_smile: 

Reverted in [revert to deconstructing](https://github.com/GCTC-NTGC/gc-digital-talent/pull/14572/commits/1241b6a894f0d64931e5408767aed0aec08b04dd)